### PR TITLE
Wazo 1102 delete refresh token

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

**Comment by pc-m** [comment]
Depends-On: https://github.com/wazo-platform/wazo-auth-client/pull/5

**Comment by pc-m** [comment]
Depends-On: https://github.com/wazo-platform/wazo-auth/pull/30

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * A new resource has been added to manage refresh tokens
 
   * GET `0.1/users/<user_uuid>/tokens`
+  * DELETE `0.1/users/<user_uuid>/tokens`
```
Comment (fblackburn1) [inline CHANGELOG.md]: The endpoint is not /users/uuid/tokens/client_id?
Same comment for swagger spec

File: integration_tests/suite/test_token.py [hunk 2]
```diff
@@ -80,6 +83,44 @@ def test_refresh_token_with_the_wrong_client_id(self):
...
+        )
+
+        with pytest.raises(HTTPError) as excinfo:
+            self.client.token.delete(user['uuid'], client_id)
+
+        assert_that(excinfo.value.response.status_code, equal_to(404))
+        assert_that(
+            excinfo.value.response.json(),
+            has_entries(
+                error_id='unknown-refresh-token',
```
Comment (fblackburn1) [inline integration_tests/suite/test_token.py]: Maybe `unknown-client-id` should be more relevant

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
regate

**Comment by fblackburn1** [comment]
regate

**Comment by fblackburn1** [comment]
regate

**Pull request merged**